src/styleguide/routes/forms.vue

```diff
@@ -106,6 +107,11 @@
           </e-label>
         </e-fieldset>
 
+        <e-fieldset legend="Date Pickers">
+          <eDate v-model="form.date" name="date" />
+          <cDatePicker :start.sync="form.date" name="date-picker" @change="onDatePickerChange" />
+        </e-fieldset>
+
```

Comment:
todo: please add some vertical spacing between the elements. Most likely you can just define `display: grid; gap: 20px;` on the `e-fieldset`.